<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>webcamClass</class>
 <widget class="QWidget" name="webcamClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>webcam</string>
  </property>
  <property name="windowIcon">
   <iconset resource="webcam.qrc">
    <normaloff>:/webcam/img/webcam.ico</normaloff>:/webcam/img/webcam.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 按钮样式 */
QPushButton:hover

 { 
	background-color: rgb(90, 90, 90);
border: none;
border-radius:5px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }
QPushButton:!hover
 { 
background-color: rgb(145, 145, 145);
 border: none;
border-radius:5px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }



/* 顶部样式 */
#XViewerClass {
    background-color: #333333;
    border-radius: 0px;
}
#headwidget
{
background-color:#3c3c3c;
}
#logo
{
	background-image: url(:/webcam/img/logo_150_40.png);
}
#close
{
background-image: url(:/webcam/img/close.png);
}
#min
{
background-image: url(:/webcam/img/min.png);
}
#max
{
background-image: url(:/webcam/img/max.png);
}
#normal
{
background-image: url(:/webcam/img/normal.png);
}
/* body 演示*/
#bodywidget
{
background-color: #212121;
}
#leftwidget
{
	background-color: #252525;
}
#camswidget
{
background-color: #1e1e1e;
}


#status
{
	color: rgb(255, 255, 255);
	font: 75 15pt &quot;Adobe Arabic&quot;;
}


/*tab 菜单按钮切换样式 */

#btn_preview
{
background-color: rgb(50, 50, 50);
}
#btn_playback
{
background-color: rgb(50, 50, 50);
}

#btn_preview:checked
{
background-color: rgb(0, 102, 184);
}
#btn_playback:checked
{
background-color: rgb(0, 102, 184);
}</string>
  </property>
  <widget class="QWidget" name="headwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>60</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="logo" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>151</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="buttonwidget" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>15</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QPushButton" name="min">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60, 60, 60);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="close">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>15</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60, 60, 60);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="max">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>15</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60, 60, 60);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="normal">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>15</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60, 60, 60);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>min</zorder>
    <zorder>close</zorder>
    <zorder>normal</zorder>
    <zorder>max</zorder>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>监控中...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_preview">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>35</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>预览</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_playback">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>35</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>回放</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="bodywidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>800</width>
     <height>540</height>
    </rect>
   </property>
   <widget class="QWidget" name="leftwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>541</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="add_cam">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="del_cam">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="change_cam">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QListWidget" name="cams_list">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="camswidget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>600</width>
      <height>540</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="playbcakwidget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>601</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="time_list">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>15:00:59</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="Calendar" name="cal"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Calendar</class>
   <extends>QCalendarWidget</extends>
   <header>calendar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="webcam.qrc"/>
  <include location="../../../WebCamDemo/资料/第九章代码XView0.1包含发布文件/src/xviewer/xviewer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>webcamClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>782</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>790</x>
     <y>-70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max</sender>
   <signal>clicked()</signal>
   <receiver>webcamClass</receiver>
   <slot>MaxWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>-57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min</sender>
   <signal>clicked()</signal>
   <receiver>webcamClass</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>705</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>-78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>normal</sender>
   <signal>clicked()</signal>
   <receiver>webcamClass</receiver>
   <slot>NormalWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_preview</sender>
   <signal>clicked()</signal>
   <receiver>webcamClass</receiver>
   <slot>Preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>-43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_playback</sender>
   <signal>clicked()</signal>
   <receiver>webcamClass</receiver>
   <slot>PlayBack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>-78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_cam</sender>
   <signal>clicked()</signal>
   <receiver>webcamClass</receiver>
   <slot>AddCam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>-50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>del_cam</sender>
   <signal>clicked()</signal>
   <receiver>webcamClass</receiver>
   <slot>DelCam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>-76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>change_cam</sender>
   <signal>clicked()</signal>
   <receiver>webcamClass</receiver>
   <slot>SetCam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>-54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cams_list</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>webcamClass</receiver>
   <slot>SelectCam(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>-10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cal</sender>
   <signal>clicked(QDate)</signal>
   <receiver>webcamClass</receiver>
   <slot>SelectDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>-19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>time_list</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>webcamClass</receiver>
   <slot>PlayVideo(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>-31</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>MaxWindow()</slot>
  <slot>NormalWindow()</slot>
  <slot>AddCam()</slot>
  <slot>DelCam()</slot>
  <slot>SetCam()</slot>
  <slot>Preview()</slot>
  <slot>PlayBack()</slot>
  <slot>SelectCam(QModelIndex)</slot>
  <slot>SelectDate(QDate)</slot>
  <slot>PlayVideo(QModelIndex)</slot>
 </slots>
</ui>
